common/locales/en/front.json

```diff
@@ -126,6 +126,7 @@
     "motivate": "Motivate yourself and your team!",
     "motivate1": "Motivate yourself to do anything.",
     "motivate2": "Get Organized. Get Motivated. Get Gold.",
+    "oldNews": "Old News (Updates)",
```

Comment:
That would make sense
On Tue, Sep 6, 2016 at 9:01 PM Alys <notifications@github.com> wrote:

> In common/locales/en/front.json
> <https://github.com/HabitRPG/habitrpg/pull/7976#discussion_r77749814>:
>
> > @@ -126,6 +126,7 @@
> >      "motivate": "Motivate yourself and your team!",
> >      "motivate1": "Motivate yourself to do anything.",
> >      "motivate2": "Get Organized. Get Motivated. Get Gold.",
> > +    "oldNews": "Old News (Updates)",
>
> I'm not sure "(Updates)" is needed, and it might be confusing - for
> example, it could be taken to mean that there are some old Bailey messages
> that have been updated with new information. @MathWhiz
> <https://github.com/MathWhiz> How would you feel about just "Old News"?
>
> —
> You are receiving this because you were mentioned.
>
>
> Reply to this email directly, view it on GitHub
> <https://github.com/HabitRPG/habitrpg/pull/7976/files/b03aecc7e9bb65f57ae7a2597baabdc72bdfadfe#r77749814>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ALz_84Ypb_wXijjXEGsYEae6R05-MiJoks5qnhrygaJpZM4J2bBN>
> .
>